### DISCUSSION TITLE
Questionb About Code Interpreter

### DISCUSSION THREAD

**Discussion by Stephen-SMJ** [zai-org/ChatGLM3#1130] [Q&A] [answered]
Hello, I would like to ask two questions. after the code interpreter of chatglm3, is it possible to continue the execution next time with the same variables as the last time?
And is there any reference to the code interpreter function as a separate API service?

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
> And is there any reference to the code interpreter function as a separate API service?
No
> after the code interpreter of chatglm3, is it possible to continue the execution next time with the same variables as the last time?
Set do_sample = False and use same seed
